cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/EntityOperatorReconciler.java

```diff
@@ -427,6 +457,10 @@ protected Future<Void> deployment(boolean isOpenShift, ImagePullPolicy imagePull
             Annotations.annotations(deployment.getSpec().getTemplate()).put(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_CERT_GENERATION, String.valueOf(caCertGeneration));
             int caKeyGeneration = clusterCa.caKeyGeneration();
             Annotations.annotations(deployment.getSpec().getTemplate()).put(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_KEY_GENERATION, String.valueOf(caKeyGeneration));
+            
+            if (unidirectionalTopicOperator && isCruiseControlEnabled && entityOperator.topicOperator() != null) {
```

Comment:
It's seems a bit odd that we check the `unidirectionalTopicOperator condition before the `entityOperator.topicOperator() != null` condition: If there's no topic operator it doesn't really make sense to talk about it being the unidirectional one.

Reply:
Thanks for spotting this.